oap-server/server-library/library-integration-test/src/main/java/org/apache/skywalking/oap/server/library/it/ITVersions.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.library.it;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * ITVersions is a utility class to read the version information from the environment file.
+ */
+public class ITVersions {
+
+    static Map<String, String> env = readingEnv("test", "e2e-v2", "script", "env");
```

Comment:
Is this work in sub folder? This seems a path to project root, but we could run this IT in maven sub module path, or separately in ide.
Have you verified these cases?

Reply:
ITVersions will read the `user.dir` to get the current working directory, then find the directory where `mvnw` is located.